### PR TITLE
Remove lpsolve files from external

### LINKED ISSUE #141: Remove lpsolve external libraries
**Is your feature request related to a problem? Please describe.**
External libraries consume space in the repository and have to be updated manually.

**Describe the solution you'd like**
Remove `lpsolve` from `external` folder and download them in `cmake` before building the package or whenever needed. 

See how this was done for eigen https://github.com/GeomScale/volume_approximation/pull/131

### PR DESCRIPTION
**Pull request by vaithak**
Now lpSolve's files will be downloaded at build time. Closes #141

### CODE REVIEW

File: cran_gen/genCRANpkg.R [hunk 1]
```diff
@@ -84,7 +129,7 @@ makevarswin_dir = paste0(path,'/cran_gen/Makevars.win')
 makevarswin_dist = paste0(path, '/cran_gen/cran_package/src')
 file.copy(makevarswin_dir, makevarswin_dist, recursive=TRUE)
 
-# replace the volume.h 
+# replace the volume.h
```
Comment (vissarion) [inline cran_gen/genCRANpkg.R]: since you edited it why not removing ?

Comment (vaithak) [reply cran_gen/genCRANpkg.R]: most probably this change occurred because I have enable automatic formatter on my editor. removed it though 👍🏽

**Comment by vissarion** [comment]
This is a great PR, it simplifies a lot our codebase. Thanks a lot!

File: cran_gen/genCRANpkg.R (hunk 1 above)
Comment (vissarion) [reply cran_gen/genCRANpkg.R]: thanks, please clean up also the other few automatic formatting that are irrelevant to this PR.

Comment (vaithak) [reply cran_gen/genCRANpkg.R]: updated 👍🏽

**Review by TolisChal — APPROVED** [review]

**Pull request merged**